gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -496,6 +548,14 @@ gst_tensor_converter_get_property (GObject * object, guint prop_id,
     case PROP_SILENT:
       g_value_set_boolean (value, self->silent);
       break;
+    case PROP_MODE:
+    {
+      gchar *mode_str =
+          g_strdup_printf ("%s:%s", self->mode, self->mode_option);
+      g_value_set_string (value, mode_str);
```

Comment:
use g_value_take_string()  / check condition (mode and mode-option is not null)

Reply:
Thanks. Changed